plugins/foreman_remote_execution/1.3/index.md

```diff
@@ -0,0 +1,877 @@
+    <th>Foreman</th>
+    <th>foreman_remote_execution</th>
+  </tr>
+  <tr>
+    <td>1.9.x</td>
+    <td>0.0.x</td>
+  </tr>
+  <tr>
+    <td>1.10.x</td>
+    <td>0.1.x - 0.2.x</td>
+  </tr>
+</table>
+
+### 1.2.2 smart_proxy_remote_execution_ssh
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Smart Proxy</th>
+    <th>smart_proxy_remote_execution_ssh</th>
+    <th>Notes</th>
+  </tr>
+  <tr>
+    <td>1.9.x</td>
+    <td>0.0.x</td>
+    <td>No compatibility with EL6 or Ubuntu 12.04</td>
+  </tr>
+  <tr>
+    <td>1.10.x</td>
+    <td>0.0.x</td>
+    <td>No compatibility with EL6 or Ubuntu 12.04</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+## 2.1 Plugin Installation
+
+This chapter covers a clean Foreman and plugin installation. We assume the
+smart proxy will be installed on the same host, however it's not a requirement.
+Please adapt according to your needs.
+
+Start by installing the Foreman nightly or latest release repository and EPEL7, see
+[Quickstart instructions](/manuals/latest/index.html#2.Quickstart) and [Foreman
+manual](/manuals/latest/index.html#3.InstallingForeman) for more information.
+
+### With Foreman Installer
+
+You can install both the Foreman and Smart Proxy plugin with the installer:
+
+    foreman-installer --enable-foreman-plugin-remote-execution\
+                      --enable-foreman-proxy-plugin-remote-execution-ssh
+
+The installer will automatically configure an SSH key for you, or you may use an existing see.
+See the `foreman-installer --help` for more information.
+
+### Manual Installation
+
+In the case you want to install the package manually on an existing Foreman or
+Katello, use the following steps.
+
+Install the packages appropriate for your operating system:
+
+RPM:
+
+    yum install tfm-rubygem-foreman_remote_execution \
+                tfm-rubygem-foreman_remote_execution_core \
+                tfm-rubygem-foreman-tasks \
+                tfm-rubygem-foreman-tasks-core \
+                rubygem-smart_proxy_dynflow \
+                tfm-rubygem-smart_proxy_dynflow_core \
+                rubygem-smart_proxy_remote_execution_ssh
+
+DEB:
+
+    apt-get install ruby-foreman-remote-execution\
+                    ruby-smart-proxy-remote-execution-ssh
+
+The next thing to do is make sure the Smart Proxy, Smart Proxy Dynflow Core
+and Foreman can talk to each other. If you're on Debian, there's no need to do anything.
+This is done by properly configuring `:core_url` in `/etc/foreman-proxy/settings.d/dynflow.yml` and `:foreman_url`, `:listen`
+and `:port` keys in `/etc/smart_proxy_dynflow_core/settings.yml`.
+
+Next you have to setup ssh keys. By default smart proxy loads the key
+from `/usr/share/foreman-proxy/.ssh/id_rsa_foreman_proxy`. To customize it you
+can edit the configuration in
+`/etc/foreman-proxy/config/settings.d/remote_execution_ssh.yml`. Without
+customization you need to create new ssh key and distribute it to
+target hosts. The key must not use passphrase.
```

Comment:
I think better would be something like "...must not be protected by a passphrase."